templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,16 @@
-# Press Enter to send
+# Enable or disable Press Enter to send
 
 By default, Zulip sends your message when you press enter (or return)
```

Comment:
I've updated the doc with `<kbd>ENTER</kbd>/<kbd>RETURN</kbd>`; it looks a bit odd to me and might do better with `Enter`/`Return`, but let me know what you think.